program/shinyApp/R/enrichment_analysis/ui.R

```diff
@@ -48,7 +48,7 @@ geneset_panel_UI <- function(
           radioGroupButtons(
             inputId = ns("file_ext"),
             label = "File Type:",
-            choices = c(".png", ".tiff", ".pdf"),
+            choices = c(".png", ".tiff", ".pdf", ".svg"),
```

Comment:
```suggestion
            choices = c(".png", ".tiff", ".svg", ".pdf"),
```
I found it ever so slightly irritating that the default value was neither left or right 🙈

Reply:
If changed, also change in the other cases